src/app/timeseries-menu.component.html

```diff
@@ -14,15 +14,26 @@ <h2 mat-dialog-title>Time Series</h2>
       [showXAxisLabel]="showXAxisLabel"
       [showYAxisLabel]="showYAxisLabel"
       [xAxisLabel]="xAxisLabel"
-      [yAxisLabel]="yAxisLabel"
+      [yAxisLabel]="yValTitle()"
       [autoScale]="autoScale"
     >
     </ngx-charts-line-chart>
     <br />
     <button mat-raised-button (click)="createCsvFromTimeseriesData()">
       Download Data
     </button>
+    <br />
+    <p>Latitude: {{ this._model.settings.CurrGridBoxLat }}</p>
+    <p>Longitude: {{ this._model.settings.CurrGridBoxLon }}</p>
+    <br />
+    <br />
+    <button mat-raised-button (click)="closeTimeSeries()">
+      Close Time Series
+    </button>
   </mat-dialog-content>
+  <h2 mat-dialog-title style="text-align: center">
+    <u>Pressure Layers</u>
```

Comment:
This one addition (_Pressure Layers_ text) seems to be messing up the size of the pop-up window and adding arrow bars to it (to scroll up and down the window).